README.md

```diff
@@ -117,10 +117,40 @@ use YieldStudio\NovaGoogleAutocomplete\AddressMetadata;
 use YieldStudio\NovaGoogleAutocomplete\GoogleAutocomplete;
 
 // This autocomplete field will return results that match a business name instead of address.
-// All the same address data is still stored.  
+// All the same address data is still stored.
 GoogleAutocomplete::make('Address')->placeType('establishment');
 ```
 
+### Capturing JSON response
+
+If you want to capture the entire response as a JSON object, you can use the `fromResponse()` helper instead of using `fromValues()`.
+This helper will capture the entire response containing all the requested values as a JSON string.
+
+```php
+// Autocompelte field
+GoogleAutocomplete::make('Location')
+    ->countries('BE')
+    ->withValues(['latitude', 'longitude', 'street_number', 'route', 'locality', 'administrative_area_level_1', 'country', 'postal_code'])
+    ->withMeta(['asJson' => true]),
+
+// Field that will capture de response object
+AddressMetadata::make('Address')
+    ->fromResponse()
+    ->invisible()
+    ->onlyOnForms()
+    ->resolveUsing(function () {
+        return json_encode(optional($this->resource)->address);
+    })
+    ->fillUsing(function ($request, $model, $attribute, $requestAttribute) {
+        if ($request->exists($requestAttribute)) {
+            $model->{$attribute} = json_decode($request[$requestAttribute], true);
+        }
+    }),
```

Comment:
This can be directly added by fromResponse, what do you think ?

Reply:
Sure, I added it. I removed the bool argument as it wouldn't make sens to call it with `false` if we enforce the resolve & fill callbacks.